<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>文件加密保护</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>软件加解密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="softAESpushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="soft3DESpushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3DES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="softDESpushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="softZUCpushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ZUC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="softRSApushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RSA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="softRC4pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RC4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="softSM4pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SM4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="softSM2pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SM2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="softSM3pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SM3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>硬件加解密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hardAESpushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hardDESpushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hard3DESpushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3DES</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
